+uudeview (0.5.13-2.1) unstable; urgency=low
...
+  * Added man page for uuwish(1).
...
+ -- Roland Rosenfeld <roland@debian.org>  Thu, 16 Sep 1999 18:48:47 +0200


+Thu Oct 24 22:12:01 1996  Martin Schulze  <joey@finlandia.infodrom.north.de>
...
+	* Corrected some manpages

Index: uudeview-0.5.20/man/uudeview.1
===================================================================
--- uudeview-0.5.20.orig/man/uudeview.1
+++ uudeview-0.5.20/man/uudeview.1
@@ -164,6 +164,16 @@ delivered in have different subject line
 verbosity. Normally, the program prints some status messages
 while reading the input files, which can be very helpful if something
 should go wrong. Use if these messages disturb you.
+Disables progress bars. See
+.B -n
+option.
+.TP
+.B -v
+(disables Verbosity) Disables verbose messages, i.e. notes are not
+displayed, but does not remove warnings and errors. Is not as quiet as
+the
+.B -q
+(Quiet) option.
 .TP
 .B -n
 No progress bars. Normally, UUDeview prints ASCII bars crawling up
Index: uudeview-0.5.20/man/uuwish.1
===================================================================
--- /dev/null
+++ uudeview-0.5.20/man/uuwish.1
@@ -0,0 +1,45 @@
+.\" Copyright (c) 1999  Roland Rosenfeld <roland@spinnaker.de>
+.\" changes Copyright (c) 2001  Chris Hanson <cph@debian.org>
+.\"
+.\" This program is free software; you can redistribute it and/or modify
+.\" it under the terms of the GNU General Public License as published by
+.\" the Free Software Foundation; either version 2 of the License, or
+.\" (at your option) any later version.
+.\"
+.\" This program is distributed in the hope that it will be useful,
+.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\" GNU General Public License for more details.
+.\"
+.\" You should have received a copy of the GNU General Public License
+.\" along with this program; if not, write to the Free Software
+.\" Foundation, Inc.,59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+.\"
+.\" This manual page is written especially for Debian Linux.
+.\"
+.TH UUWISH 1 "February 2001" "Debian Project" "Debian GNU/Linux"
+.SH NAME
+uuwish \- A minimal wish extended by the UU commands
+.SH SYNOPSIS
+.B uuwish
+is a minimal
+.BR wish (1)
+extended by the UU commands that are available in
+.BR uudeview (1)
+and
+.BR uuenview (1).
+.B uuwish
+isn't meant to be called directly; it is a Tcl/Tk script interpreter,
+which is meant to be used by
+.BR xdeview (1).
+.SH "SEE ALSO"
+.BR xdeview (1),
+.BR uudeview (1),
+.BR uuenview (1),
+.BR wish (1).
+.SH AUTHOR
+.B uuwish
+was written by Frank Pilhofer <fp@informatik.uni-frankfurt.de>.
+.PP
+This manual page was written by Roland Rosenfeld <roland@debian.org>,
+for the Debian GNU/Linux system (but may be used by others).
Index: uudeview-0.5.20/configure.in
===================================================================
--- uudeview-0.5.20.orig/configure.in
+++ uudeview-0.5.20/configure.in
@@ -607,7 +607,7 @@ if test "x$have_minews" = "xyes" ; then
 fi
 if test "x$have_tk" != "xno" ; then
 	PROGS="$PROGS xdeview"
-	MPAGES="$MPAGES xdeview.1"
+	MPAGES="$MPAGES xdeview.1 uuwish.1"
 	DOINST="$DOINST install-tcl"
 fi
 
